component/src/main/java/io/siddhi/extension/io/file/FileSource.java

```diff
@@ -265,7 +265,28 @@
                                 "Once file content is completely read, " +
                                 "it will keep checking whether a new entry is added to the file or not.\n" +
                                 "If such entry is added, it will be immediately picked up and processed.\n"
-                )
+                ),
+                @Example(
+                        syntax = "" +
+                                "@source(type='file',\n" +
+                                "mode='text.full',\n" +
+                                "tailing='false'\n " +
+                                "dir.uri='file://abc/xyz',\n" +
+                                "action.after.process='delete',\n" +
+                                "@map(type='csv' @attributes(eof = 'trp:eof'))) \n" +
+                                "define stream FooStream (symbol string, price float, volume long, eof boolean); \n",
+
+                        description = "" +
+                                "Under above configuration, all the files in directory will be picked and read " +
```

Comment:
@Methma shall we rephrase the description?
contains valid json with "keys 'symbol', 'price' , 'volume'. (removed duplicated `json`. No need to have "'eof" since we are creating the eof internally.)
...
its content will be converted to an event using siddhi-map-json extension with additional `eof` attribute
...